setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+setup(
+    name='AMP-CLI',
+    version='0.1',
+    py_modules=['amp_cli'],
+    install_requires=[
+        'click',
+        'docker',
```

Comment:
```suggestion
        'docker',
        'wheel`,
```
I also had to `pip install wheel` for this to build properly. So we may want to add that here.

Reply:
Gotcha